### PR TITLE
Add FA Updates Summer 2021 news post

### PR DESCRIPTION
**Pull request by pishifat**
!!!

### CODE REVIEW

File: news/2021-06-30-featured-artist-track-updates-summer-2021.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+
+New songs are also listed below along with any relevant ranked maps and an audio preview or two. These should give you a quick impression of what's in store, but also remember that all song previews are available from each Featured Artist page!
+
+---
+
+## [BilliumMoto / Dictate / Kurokotei](https://osu.ppy.sh/beatmaps/artists/45)
+
+[![](https://assets.ppy.sh/artists/38/header.jpg)](https://osu.ppy.sh/beatmaps/artists/38)
+
+Coming as a surprise to nobody, we've officially added [**TWC Sound Team "Zeroth Powers" - One One**](https://osu.ppy.sh/beatmapsets/1422682) from the TWC world cup grand finals to the Featured Artist listings for [BilliumMoto](https://osu.ppy.sh/users/3862471), [Dictate](https://osu.ppy.sh/beatmaps/artists/21), and [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58)! Maybe we'll have a listing for *Johnson Solid* one day too.
```
Comment (zeusminus) [inline news/2021-06-30-featured-artist-track-updates-summer-2021.md:23]:
```suggestion
Coming as a surprise to nobody, we've officially added [**TWC Sound Team "Zeroth Powers" - One One**](https://osu.ppy.sh/beatmapsets/1422682) from the [osu!taiko World Cup 2021](/wiki/Tournaments/TWC/2021) Grand Finals to the Featured Artist listings for [BilliumMoto](https://osu.ppy.sh/users/3862471), [Dictate](https://osu.ppy.sh/beatmaps/artists/21), and [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58)! Maybe we'll have a listing for *Johnson Solid* one day too.
```
"TWC world cup" is an incorrect expression here, as "TWC" already contains the phrase "world cup". Also linking to the respective tourney wiki would be nice, but not mandatory.

File: news/2021-06-30-featured-artist-track-updates-summer-2021.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/31/Tera%20I_O/Camellia%20-%20Tera%20IO.mp3" type="audio/mpeg">
+</audio>
+
+## [Frums](https://osu.ppy.sh/beatmaps/artists/68)
+
+[![](https://assets.ppy.sh/artists/68/header.jpg)](https://osu.ppy.sh/beatmaps/artists/68)
+
+And we can't forget the osu!catch world cup! [**Frums - of Ambrosia**](https://osu.ppy.sh/beatmapsets/1478486) was debuted as the tournament's grand finals tiebreaker. Like *Camellia - ΩΩPARTS* and *TWC Sound Team "Zeroth Powers" - One One*, this track was specifically created for the world cups — a trend you'll likely see going forward!
```
Comment (zeusminus) [inline news/2021-06-30-featured-artist-track-updates-summer-2021.md:108]:
```suggestion
And we can't forget the [osu!catch World Cup](/wiki/Tournaments/CWC/2021)! [**Frums - of Ambrosia**](https://osu.ppy.sh/beatmapsets/1478486) was debuted as the tournament's grand finals tiebreaker. Like *Camellia - ΩΩPARTS* and *TWC Sound Team "Zeroth Powers" - One One*, this track was specifically created for the world cups — a trend you'll likely see going forward!
```

File: news/2021-06-30-featured-artist-track-updates-summer-2021.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+- **Camellia - Newspapers for Magicians**
+  - [osu! beatmap](https://osu.ppy.sh/beatmapsets/938426) by [Mir](https://osu.ppy.sh/users/8688812)
+  - [osu!mania beatmap](https://osu.ppy.sh/beatmapsets/929143) by [Leniane](https://osu.ppy.sh/users/7138602)
+- **Camellia - Qyoh (Nine Stars)**
+- **Camellia - WE ARE \*NOT\* CURZED**
+- **Camellia & DJ Genki - Feelin Sky**
+  - [osu! beatmap](https://osu.ppy.sh/beatmapsets/366706) by [EphemeralFetish](https://osu.ppy.sh/users/4433793)
+  - [osu! beatmap](https://osu.ppy.sh/beatmapsets/414543) by [Strategas](https://osu.ppy.sh/users/2971837)
+  - [osu! beatmap](https://osu.ppy.sh/beatmapsets/882932) by [adioni](https://osu.ppy.sh/users/8894794)
+  - [osu! beatmap](https://osu.ppy.sh/beatmapsets/524256) by [Curiossity](https://osu.ppy.sh/users/6490035). Yeah, there's four in the same mode. Variety is for chumps.
```
Comment (AncuL001) [inline news/2021-06-30-featured-artist-track-updates-summer-2021.md:63]: this one is mania

**Review by Ephemeralis — APPROVED** [review]

File: news/2021-06-30-featured-artist-track-updates-summer-2021.md [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/32/TRAUMA/onumi%20X%20Fluff%20Pink%20%26%20Belak%20-%20TRAUMA.mp3" type="audio/mpeg">
+</audio>
+
+## [Station Earth / Blue Marble](https://osu.ppy.sh/beatmaps/artists/66)
+
+[![](https://assets.ppy.sh/artists/66/header.jpg)](https://osu.ppy.sh/beatmaps/artists/66)
+
+[*Station Earth - Cold Green Eyes feat. Roos Denayer](https://osu.ppy.sh/beatmapsets/371128) is an osu! classic at this point. The song's creator **Blue Marble** (previously known as **Station Earth**) re-designed the track in a whole new way, so we've added it to their Featured Artist listing!
```
Comment (AncuL001) [inline news/2021-06-30-featured-artist-track-updates-summer-2021.md:270]: missing leading asterisk

**Pull request merged**